src/l10n.json

```diff
@@ -184,6 +185,7 @@
     "registration.studentUsernameStepHelpText": "Already have a Scratch account?",
     "registration.studentUsernameStepTooltip": "You'll need to create a new Scratch account to join this class.",
     "registration.studentUsernameFieldHelpText": "For safety, don't use your real name!",
+    "registration.acceptTermsOfService": "By creating an account, I accept and agree to the Terms of Service.",
```

Comment:
I thought we were going to link to the TOS here.

Reply:
Added a link!